tutorials/token-gating-tutorial.md

```diff
@@ -0,0 +1,259 @@
+---
+sidebar_position: 1
+sidebar_label: Token Gating Tutorial
+slug: /token-gating
+---
+
+# Collab.Land Token Gating Tutorial
+
+Collab.Land offers a token gating feature that allows you to restrict access to elements or pages of your site based on ownership of certain blockchain assets. This tutorial walks you through creating your own token gate for your website using the Collab.Land API.
+
+### Website Gating Logic Flow
+
+1. Define Token Gating Rules (TGRs) for your website.
+2. User visits your website.
+3. Collect user's wallet address.
+4. Call Collab.Land API to check if the user has the required tokens to fulfil your rules.
+5. Display gated content if the user has the required tokens.
+
+![TGR Workflow](imgs/tgr-flow.png)
+To check if a user has the required tokens via the Collab.Land API, you need:
+
+- The user's wallet address.
+- Token Gating Rules for your site.
+
+This tutorial implements the gating logic by calling the Collab.Land [`access-control/check-roles`](../docs/downstream-integrations/api/token-gating.md) endpoint with the user's wallet address and the [Token Gating Rules](#define-token-gating-rules) as the payload for the request. Collab.Land returns `true/false` for each rule to indicate if the given address fits the provided criteria or not.
```

Comment:
`downstream-integrations` references are broken in the live version. how to fix @kenny-io?

Reply:
sorry, wrong original author